mentor_classifier/mentor_classifier/api.py

```diff
@@ -99,6 +99,35 @@ def get_off_topic_threshold() -> float:
 }
 """
 
+GQL_QUERY_MENTOR_ME = """
+query Mentor{
+    me {
+        mentor {
+            name
+            answers {
+                _id
+                question {
+                    _id
+                    question
+                    paraphrases
+                    type
+                    name
+                    mentor
+                    mentorType
+                    minVideoLength
```

Comment:
I don't think we actually need a generic "get everything for a mentor" query at all?

Can we change this to something like 

```python
GQL_QUERY_MENTOR_ANSWERS_AND_NAME = """
query Mentor{
    me {
        mentor {
            name
            answers {
                question {
                    question
                }
                transcript
        }
}
```

and then

```python
def fetch_mentor_answers_and_name() -> Tuple[List[AnswerInfo], str]:
   # use the above query and organize the result
   #  into a list of AnswerInfo and the mentor's name
```